test/codegen/golden/expected/auth_service/app/routers/users.py

```
@@ -14,47 +17,47 @@
 
 @router.post("/auth/register", status_code=201)
 async def register(
-    body: UserCreate,
+    body: RegisterRequest,
     session: AsyncSession = Depends(get_session),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)